subsys/mpsl/Kconfig

```diff
@@ -18,4 +19,28 @@ config MPSL_SIGNAL_STACK_SIZE
 	  Size of the signal handler thread stack, used to process lower
 	  priority signals in MPSL.
 
+choice CLOCK_CONTROL_NRF_SOURCE
```

Comment:
The clock source choice and configs are not necessary here, so I will remove those. Will be taken from zephyr/drivers/clock_control/Kconfig.nrf. Only CLOCK_CONTROL_NRF_ACCURACY is required in the mpsl subsys Kconfig.